db/seeds.rb

```diff
@@ -186,3 +186,11 @@
   r.location = boulder
   r.feed_format = "video"
 end.save!
+
+if Rails.env.development?
```

Comment:
I would like to create an admin user in production also. Maybe something like this:

  AdminUser.create!(
    email: ENV['ADMIN_EMAIL']
    password: ENV['ADMIN_PASSWORD'],
    password_confirmation: ENV['ADMIN_PASSWORD']
  )